### PR TITLE
filter community contributions

### PR DESCRIPTION
**Pull request by sarveshpro**

### CODE REVIEW

File: src/TreeMap/App.svelte [hunk 1]
```diff
@@ -82,90 +82,181 @@
...
+                ["to-number", ["get", "treeCount"]],
+                { "max-fraction-digits": 1 },
+              ],
+              "text-font": ["Ubuntu Bold"],
+              "text-size": 12,
+            },
+            paint: {
+              "text-color": "#fff",
+            },
+          });
```
Comment (norbertschuler) [inline src/TreeMap/App.svelte:170]: Is it really necessary to do this code twice. Can't we just make an if clause for the following?
```
        if (community === "true") {
          const filteredContributions = contributions;
        } else {
          const filteredContributions = contributions.filter((contrib) => {
            return contrib.properties.type !== "gift";
          });
        }
```

**Review by norbertschuler — CHANGES_REQUESTED** [review]
Please remove unnecessary cod duplication.

File: src/TreeMap/App.svelte (hunk 1 above)
Comment (sarveshpro) [reply src/TreeMap/App.svelte]: Didn't work when I did it ;) done now.

**Review by norbertschuler — APPROVED** [review]
LGFM

**Pull request merged**